package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0-semantically-released",
   "description": "Custom jest matchers to test the state of the DOM",
   "main": "dist/index.js",
+  "types": "extend-expect.d.ts",
```

Comment:
The problem with this is is that the `main` file, `dist/index.js`, does not extend the global `expect` object. So only doing `import @testing-library/jest-dom` will provide typings for the extensions but it will fail at runtime because `expect` was not extended.

Reply:
OK, sure. So we can instead use a folder. Let me test that.

Reply:
@jgoz This has been updated, and is still working for me locally.